framework/doc/content/source/actions/ReadExecutorParamsAction.md

```diff
@@ -1,21 +1,8 @@
 # ReadExecutorParamsAction
 
-!alert construction title=Undocumented Action Class
-The ReadExecutorParamsAction has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with an Action;
-however, what is contained is ultimately determined by what is necessary to make the documentation
-clear for users.
-
 !syntax description /Executors/ReadExecutorParamsAction
 
-## Overview
-
-!! Replace these lines with information regarding the ReadExecutorParamsAction action.
-
-## Example Input File Syntax
-
-!! Describe and include an example of how to use the ReadExecutorParamsAction action.
-
-!syntax description /Executors/ReadExecutorParamsAction
+This [MooseObjectAction](MooseObjectAction.md) act on the `[Executor]` block
+to create Executor objects. See [syntax/Executors/index.md] for more information.
```

Comment:
We have the `AddExecutorAction` to create Executor objects though, right? This just reads in the executor options and adds automatic preconditioning if required, it looks like. If this is the case, then the source class description might need some updating too.

Reply:
Really weird how you couldn't get the markdown syntax to work with `AddExectorAction`. I suppose it's enough of a base class that MooseDocs doesn't fully register it. Regardless I like your new changes.

Reply:
yeah. I tried to get the automatic stub generation to get it right but it didnt work on my machine. I ll try again somewhere else another time. Thanks for the review!